src/meta.rst

```diff
@@ -71,39 +71,128 @@ A full description of selectors is
 `in the conda docs <http://conda.pydata.org/docs/building/meta-yaml.html#preprocessing-selectors>`_.
 
 
-Using the Compilation ``toolchain``
------------------------------------
-Packages that need to compile code should add the ``toolchain`` package as a build dependency. This helps set up environment variables to instruct the compilers to do the right thing (e.g. building against the right version of the MacOS SDK).
+Pinning packages
+----------------
+
+Conda-smithy 3.0.0 switches to ``conda-build=3``. In conda-smithy 3.0.0, we use a central configuration file from conda-forge-pinning for the build matrices and versions of specific packages.
```

Comment:
"conda-forge-pinning" -> "``conda-forge-pinning``"

Also might be worth adding a link.